DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests.xcodeproj/project.pbxproj

```diff
@@ -549,7 +549,7 @@
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/ncreated/Framing";
 			requirement = {
-				branch = framer;
+				branch = "ship-framer";
```

Comment:
Pointing to stable branch in https://github.com/ncreated/Framing - I added basic text alignment options in there, so we can preview `SRTextPosition` in snapshot tests.